scripts/_common.sh

```diff
@@ -1,31 +1,15 @@
 #!/bin/bash
 
 #=================================================
-# PERSONAL HELPERS
+# COMMON VARIABLES
 #=================================================
 
+app_depencencies="default-jre-headless daemon net-tools"
```

Comment:
```suggestion
app_depencencies="openjdk-11-jdk daemon net-tools"
```

I wanted to apply #31 but the corresponding PR have some trouble, so I just applying the commit